### ISSUE TITLE
Scrolling issue with screen reader

### ISSUE THREAD

**Issue by marisademeglio** [daisy/pipeline-ui#23]
"The text in “Messages” list does not scroll with screen reader navigation"

cc @prashantverma2014

**Comment by rachanasinghg** [comment]
I am unable to navigate to Messages Window after running the pipeline script. The results are shown in Message window, but my keyboard navigation is between job name, Tab panel, 'Show Results folder' button, Epub3 and Log link. Screen reader reads these names but not the content inside the window.

**Comment by avneeshsingh** [comment]
Looking at Rachana's report, this is a critical bug for keyboard navigation.

**Comment by prashantverma2014** [comment]
The screen scrolls as I read with Jaws.
